app/src/main/java/org/sopt/cds29cm/presentation/category/CategoryHorizontalViewHolder.kt

```diff
@@ -0,0 +1,14 @@
+package org.sopt.cds29cm.presentation.category
+
+import org.sopt.cds29cm.data.CategoryHorizontal
+import org.sopt.cds29cm.databinding.ItemCategoryHorizontalBinding
+import androidx.recyclerview.widget.RecyclerView
+
+class CategoryHorizontalViewHolder(private val binding: ItemCategoryHorizontalBinding) :
+    RecyclerView.ViewHolder(binding.root)
+    {
+        fun onBind(categoryHorizontalData: CategoryHorizontal) {
+            binding.tvCategoryHorizontal.text = categoryHorizontalData.categoryName
+            binding.ivCategoryHorizontal.setImageResource(categoryHorizontalData.categoryImage)
```

Comment:
물~론 binding이 2개밖에 없어서 굳이 스코프 함수를 안써도 되지만, 저라면 습관화를 위해 이렇게 작성했을 것 같아요
```suggestion
            with(binding) {
                    tvCategoryHorizontal.text = categoryHorizontalData.categoryName
                    ivCategoryHorizontal.setImageResource(categoryHorizontalData.categoryImage)
            }
```

Reply:
저 궁금한거!! 지금까지는 종종 그냥 함수 선언하는 줄에서 함수가 binding 리턴하는거 아니여도 
`    fun onBind(categoryHorizontalData: CategoryHorizontal) =with(binding) {` 
이렇게 퉁쳐서 `=with(binding) `으로 썼었는데 리턴 안 할 때는 제시해준거 처럼 아예 줄 바꿔서 `with(binding) { ~}` 이걸루 구분해서 사용하는게 좋으려나요....?

Reply:
완전 취향차이! 근데 말씀해주신것처럼, =with(binding) 이 더 코틀린스러운 문법입니당 !! 리턴값을 지정해주는건 더 자바스러워서, 더 바람직한 방법으로 알고계시네요 ~~